addons/survey/static/src/scss/survey_form.scss

```diff
@@ -0,0 +1,251 @@
+.o_survey_header {
+    height: 50px;
+}
+.o_survey_main {
+    min-height: calc(100% - 105px);
+    padding-top: 16px;
+}
+
+.o_survey_question_text_box,
+.o_survey_question_date,
+.o_survey_question_datetime,
+.o_survey_question_numerical_box {
+    border: 0px;
+    border-radius: 0px;
+    border-bottom: 2px solid $o-brand-primary;
+    background-color: transparent !important;
+    font-weight: bold;
+    color: $o-brand-primary !important;
```

Comment:
Check if you can replace some of those with bootstrap default classes.
Such as border-0, font-weight-bold, ...

Reply:
+1